### PR TITLE
{numlib}[iompi/2020b] FFTW v3.3.8

### PR DESCRIPTION
**Pull request by DirectorSloan**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/f/FFTW/FFTW-3.3.8-iompi-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+checksums = [
+    '6113262f6e92c5bd474f2875fa1b01054c4ad5040f6b0da7c03c98821d9ae303',  # fftw-3.3.8.tar.gz
+    '1b3319b98a2ca4ead68290b3229385c0573e22749a5a2ffb49486a0bbb37dc1e',  # FFTW-3.3.8_fix-icc-no-gcc.patch
+]
+
+# no quad precision, requires GCC v4.6 or higher
+# see also
+# http://www.fftw.org/doc/Extended-and-quadruple-precision-in-Fortran.html
+with_quad_prec = False
+
```
[the remark this answers is not in the document]
Comment (DirectorSloan) [reply easybuild/easyconfigs/f/FFTW/FFTW-3.3.8-iompi-2020b.eb:23]: No there is no reason. Just took it over from the [3.3.8-iomkl-2018b](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/f/FFTW/FFTW-3.3.8-iomkl-2018b.eb)

[the remark this answers is not in the document]
Comment (DirectorSloan) [reply easybuild/easyconfigs/f/FFTW/FFTW-3.3.8-iompi-2020b.eb:23]: Nevermind, I just oversaw what you mean and mixed up the parameters `with_quad_prec = False` and `use_fma4 = False` .

Sorry and thx for the hint. Have to change this.

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn124.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/b2e433a3b7fd415bf96459727bdd2128 for a full test report.

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @DirectorSloan!

**Pull request merged**